### ISSUE TITLE
Delay JavaScript Execution - Iubenda JS error with default exclusions

### ISSUE THREAD

**Issue by vmanthos** [wp-media/wp-rocket#4217]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version ✅3.9.1.1
- Used the search feature to ensure that the bug hasn’t been reported before ✅

**Describe the bug**

When the default exclusions are in place:
```
/jquery-?[0-9.](.*)(.min|.slim|.slim.min)?.js
js-(before|after)
(?:/wp-content/|/wp-includes/)(.*)
```
a JavaScript error pops up when Iubenda's inline JavaScript is delayed:

![](https://i.vgy.me/b2vweK.jpg)

This is because Iubenda's external files are hosted locally, and are excluded from Delay JavaScript Execution by `(?:/wp-content/|/wp-includes/)(.*)`.

Excluding the following fixes the issue:
```Javascript
_iub
```

**To Reproduce**

1. Add the following in the `<head>`: https://snippi.com/raw/gk1eybm
2. Make sure the default exclusions are in place.
3. Check the JavaScript console for the error.

**Expected behavior**

No errors should be there when Delay JavaScript Execution is active.

**Additional context**

This issue will be there for all cases where:
1. External JavaScript is loaded, and that relies on inline JavaScript.
2. JavaScript minification is active, which saves files locally.
3. Default exclusions are in place.

Maybe we should consider not saving JavaScript files locally when the default exclusions are in place.

**Related ticket:** https://secure.helpscout.net/conversation/1581925155/282758/

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by NataliaDrause** [comment]
Related: https://secure.helpscout.net/conversation/1752424903/318375/

**Comment by piotrbak** [comment]
Will be fixed in the 3.13 backend

**Comment by piotrbak** [comment]
Handled in 3.13 using dynamic lists
